src/components/Company/Offers/Manage/CompanyOffersTitle.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { Chip } from "@material-ui/core";
+
+export const statusChips = {
+    hidden: <Chip label="Hidden" size="small" style={{ backgroundColor: "#90A4AE" }} />,
+    blocked: <Chip label="Blocked" size="small" style={{ backgroundColor: "#DC4338" }} />,
+    archived: <Chip label="Archived" size="small" style={{ backgroundColor: "#56A8D6" }} />,
+};
+
+const OfferTitle = ({ title, offersVisibility, setOfferVisibility, offerId }) => {
+    const [chips, setChips] = useState([]);
+    const isHidden = offersVisibility[offerId]?.isHidden;
+    const isBlocked = offersVisibility[offerId]?.isDisabled;
+    const isArchived = offersVisibility[offerId]?.isArchived;
+
+    useEffect(() => {
+        const tempChips = [];
+        if (isHidden)
+            tempChips.push(statusChips.hidden);
+        if (isBlocked)
+            tempChips.push(statusChips.blocked);
+        if (isArchived)
+            tempChips.push(statusChips.archived);
+        setChips(tempChips);
+    }, [isArchived, isBlocked, isHidden, offerId, offersVisibility, setOfferVisibility]);
+
+    return (
+        <>
+            {title}
+            <div style={{ position: "absolute" }}>
```

Comment:
Even though it is not the best solution now (MaterialUI is in version 5, we are using version 4, one of the breaking changes is related to the styling solution which changed completely) you should not have to place styles directly on elements like you did here: why not create a separate style for it?
That way extending the styles for this component can be done in a more centralized way (near other style definitions).